app/javascript/leaflet/maps/homepage_map.js

```diff
@@ -0,0 +1,57 @@
+import GeoBlacklightViewerMap from "../viewers/map";
```

Comment:
There is something kind of screwy here. Might be why the map isn't working but I am not sure.